src/trues.jl

```diff
@@ -44,10 +44,11 @@ function Base.getindex(x::AbstractArray{T,N}, mask::Trues{N, NTuple{N,Base.OneTo
     return x[trues(size(x))] # else revert to usual getindex method
 end
 
-# https://github.com/JuliaArrays/FillArrays.jl/issues/148
+# https://github.com/JuliaArrays/FillArrays.jl/issues/148 and 150
 function Base.getindex(
-    a::AbstractFill{T, 1, Tuple{Base.OneTo{Int}}},
-    b::Trues{1, Tuple{Base.OneTo{Int}}}) where T
-    @boundscheck length(a) == length(b) || throw(BoundsError(a, b))
+    a::AbstractFill{T, N, Tuple{Vararg{Base.OneTo{Int64}, N}}},
+    b::Trues{N, Tuple{Vararg{Base.OneTo{Int64}, N}}},
```

Comment:
```suggestion
    b::Trues{N, Tuple{Vararg{Base.OneTo{Int}, N}}},
```

Reply:
oops, sorry you had to suggest this again, just like my previous PR.
i keep just copying the type suggested by julia for resolving the ambiguity...